.eleventy.js

```diff
@@ -84,6 +78,12 @@ module.exports = function (config) {
         }
     });
 
+    config.on("beforeBuild", () => {
+        if (!Object.keys(siteConfig.languages).includes(siteConfig.defaultLanguage)) {
+            console.error("The default language configured in src/_data/config.json is not one of your site's supported languages."); // eslint-disable-line no-console
+        }
```

Comment:
I don't think you need to convert the `siteConfig.languages` Object to an array of keys. You could probably get away with just checking if the value is truthy. 

Something like the following might work:

```javascript
if (!siteConfig.langauges[siteConfig.defaultLanguage]) {
    throw new Error(`The default language, ${siteConfig.defaultLanguage}, configured in src/_data/config.json is not one of your site's supported languages.`);
}
```
I'm suggesting throwing an error so that there is a build time error, rather than outputting the site in a potentially invalid state. There may be another way to achieve this with 11ty.

Reply:
Gotcha— this works! Added in c5b7b64.